test/unit/blocks_control.js

```diff
@@ -261,7 +261,8 @@ test('wait', t => {
     const args = {DURATION: .01};
     const waitTime = args.DURATION * 1000;
     const startTest = Date.now();
-    const threshold = 1000 / 60; // 60 hz
+    const thresholdSmall = 1000 / 60; // only allow the wait to end one 60Hz frame early
+    const thresholdLarge = 1000 / 3; // be less picky about when the wait ends, in case CPU load makes the VM run slowly
```

Comment:
Maybe these would be better named `thresholdMin` and `thresholdMax`?

Reply:
That was my initial impulse, but `thresholdSmall` is used in two different ways: once as the amount to add to `timeEleapsed` as the failsafe to "emergency exit" the busy-wait in case `mockUtil.stackFrame.timer.timeElapsed()` isn't working correctly, and the other is as an amount to subtract from `timeElapsed` to establish the minimum allowed wait time. IMO, `thresholdMin` only fits the latter.